### ISSUE TITLE
'yy' and 'p' don't work.

### ISSUE THREAD

**Issue by neuromaancer** [m4xshen/hardtime.nvim#37]
**Describe the bug**
When I used this plugin, I found the normal yank and paste functions don't work, for example 'yy' and 'p' as follows.

**Config options**
```lua
﻿﻿  "m4xshen/hardtime.nvim",
  lazy = false,
  dependencies = { "MunifTanjim/nui.nvim", "nvim-lua/plenary.nvim" },
  opts = { -- Add "oil" to the disabled_filetypes
    disabled_filetypes = { "qf", "netrw", "NvimTree", "lazy", "mason", "oil" },
    restricted_keys = {
      ["h"] = {},
      ["j"] = {},
      ["k"] = {},
      ["l"] = {},
    },
  },
```

**Comment by m4xshen** [comment]
Are you using any Neovim distro?

**Comment by neuromaancer** [comment]
> 

LazyVim actually. But I don't think there is a conflict?

**Comment by m4xshen** [comment]
Does `dd` and then `p` work?

**Comment by neuromaancer** [comment]
> 

No, `dd` works as expected, but not the `p`.

**Comment by neuromaancer** [comment]
more info: why I said yy doesn't work, because I have set 'highlight on yank' autocmd, but yy didn't highlight the text, shift+y did.

**Comment by m4xshen** [comment]
Please try `yyp` again with the following config:
```lua
﻿﻿  "m4xshen/hardtime.nvim",
  lazy = false,
  dependencies = { "MunifTanjim/nui.nvim", "nvim-lua/plenary.nvim" },
  opts = { -- Add "oil" to the disabled_filetypes
    disabled_filetypes = { "qf", "netrw", "NvimTree", "lazy", "mason", "oil" },
    restricted_keys = {
      ["h"] = {},
      ["j"] = {},
      ["k"] = {},
      ["l"] = {},
    },
    resetting_keys = {
      ["y"] = {},
      ["p"] = {},
   }
  },
```

**Comment by m4xshen** [comment]
Also do you have any keymap related to `y` or `d`?

**Comment by neuromaancer** [comment]
> 

Yes, this works, but I think this is a compromised way to do it, right? I checked my remaps (also the LazyVim's), there is no keymap related to `y` or `d`.

**Comment by m4xshen** [comment]
Yeah this is just for debugging.

**Comment by neuromaancer** [comment]
Sorry I just find I have a plugin concerning yank and paste action, but it is inside the LazyVim, called yanky.nvim. I bet this is not the issue.

Concrete mapping: https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/plugins/extras/coding/yanky.lua

Hope that this is helpful!

**Comment by m4xshen** [comment]
Fixed in ab488fa. Thanks for the feedback!
